### ISSUE TITLE
display training and validation loss, question

### ISSUE THREAD

**Issue by Shiro-LK** [dreamquark-ai/tabnet#185]
Hi,

Thanks for sharing an implementation of tabnet, it is very usefull.

I have two question regarding the display of the training/validation loss : 
- why there is a "minus" (-) before the value of the loss ?
- why the stopping loss for regression is use MSE instead of the loss_fn chosen in fit function ?

Thank for the help !

**Comment by Optimox** [comment]
Hello @Shiro-LK,

Basically the current feature for early stopping is not so good.
You can't provide your custom metric for early stopping: for classification it's AUC, for regression it's MSE

This is a feature we need to implement.
Early stopping loss and loss function for training are not always the same, for classification it's common to validate score with AUC but you can't use AUC as a loss function. That's why they will stay to different things.

The reason you see a "-" sign in front of the MSE is a because we were lazy when implementing this (we added a "-" because for classification the stopping metric is -AUC so we wanted to display something with a positive sign, so we ended up with a negative sign for regression).

So:
- do not worry too much about the "-" sign you see
- we need to add a better early stopping method so that you can give your own early stopping metric (when doing this we will deal with the "-" sign too)

**Comment by Optimox** [comment]
@Shiro-LK this has been solved by #193, things are now much clearer, either use develop or wait for the next release in order to use it.
